lava/lib/dl/bootstrap/README.md

```diff
@@ -1,6 +1,8 @@
 # Lava-DL Bootstrap
 
-`lava.lib.dl.bootstrap` accelerates rate coded Spiking Neural Network (SNN) training using by dynamically estimating the equivalent ANN transfer function of a spiking layer with a picewise linear model at regular interval and using the ANN equivlent network to train the original SNN. 
+In general ANN-SNN conversion methods for rate based SNN result in high latency of the network. This is because the rate interpretation of a spiking neuron using ReLU acitvation unit breaks down for short inference times. 
+
+`lava.lib.dl.bootstrap` accelerates rate coded Spiking Neural Network (SNN) training by dynamically estimating the equivalent ANN transfer function of a spiking layer with a picewise linear model at regular interval and using the ANN equivlent network to train the original SNN. 
```

Comment:
by dynamically estimating the equivalent ANN transfer function of a spiking layer with a piecewise linear model at regular epoch intervals and using the ANN equivalent